src/toast/_toast.scss

```diff
@@ -0,0 +1,20 @@
+.toast {
+  position: fixed;
+  bottom: 0;
+  z-index: 2;
```

Comment:
I think we can make the `z-index` a very large number, such as `1000`, so that it will be above all content.

Reply:
We decided we can handle this on project scope. I think the initial `z-index` value can stay 2.